tests/github-plugin-diff.js

```diff
@@ -174,19 +168,13 @@ github.pullRequests.get({
     process.exit(0);
   }
 
-  return Promise.all(diffTasks.map(task => {
-    return new Promise((resolve, reject) => {
-      Promise.all(task.plugins.map(plugin => diffPluginOutputs(plugin, process.env.TRAVIS_BRANCH, task.fixtures)))
-      .then(outputs => {
-        task.outputs = outputs;
-        resolve(task);
-      });
-    });
-  }));
+  for (const task of diffTasks) {
+    task.outputs = task.plugins.map(plugin => diffPluginOutputs(plugin, process.env.TRAVIS_BRANCH, task.fixtures));
+  }
```

Comment:
No, it isn't. That's why the next Promise uses the empty parameter brackets `() =>`.